src/libpmemfile-core/out.c

```diff
@@ -39,11 +39,20 @@
 #include <stdio.h>
 #include <stdarg.h>
 #include <stdlib.h>
-#include <unistd.h>
 #include <limits.h>
 #include <string.h>
 #include <errno.h>
 
+#ifdef DEBUG
+#ifndef _WIN32
+#include <unistd.h>
+#define os_getpid getpid
+#else
+#include <process.h>
```

Comment:
Well, if have the os_thread things in a separate TU, I would prefer to do the same with this as well. Who knows what macros are defined in process.h
Perhaps we could add a os_util.h, os_util.c ?